config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    suggestion: 'Check the `cloud.log` for more information.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_CANNOT_UPDATE_VERSION:
+    step: 'deploy-static-content:generate'
+    title: 'Cannot update the deployed version'
+    suggestion: 'Cannot update `./pub/static/deployed_version.txt` file. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_CLEAN_FAILED:
+    step: clean-static-content
+    title: 'Failed to clean static content files'
+    suggestion: ''
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SPLIT_DB_COMMAND_FAILED:
+    step: 'install-update: split-db'
+    title: 'Command `/bin/magento setup:db-schema:split` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_VIEW_PREPROCESSED_CLEAN_FAILED:
+    step: clean-view-preprocessed
+    title: 'Failed to clean the `var/view_preprocessed` folder'
+    suggestion: 'Unable to clean the `./var/view_preprocessed` folder. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_FILE_CREDENTIALS_EMAIL_NOT_WRITABLE:
+    step: 'install-update: reset-password'
+    title: 'Failed to update the `/var/credentials_email.txt` file'
+    suggestion: 'Failed to update the `/var/credentials_email.txt` file. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_UPGRADE_COMMAND_FAILED:
+    step: 'install-update: update'
+    title: 'Command `/bin/magento setup:upgrade` failed'
+    suggestion: 'Check the `cloud.log` and `install_upgrade.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_CACHE_FLUSH_COMMAND_FAILED:
+    step: clean-cache
+    title: 'Command `/bin/magento cache:flush` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_MAINTENANCE_MODE_DISABLING_FAILED:
+    step: disable-maintenance-mode
+    title: 'Command `/bin/magento maintenance:disable` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_UNABLE_TO_READ_RESET_PASSWORD_TMPL:
+    step: 'install-update: reset-password'
+    title: 'Enable to read reset password template'
```

```suggestion
    title: 'Unable to read reset password template'
```

Not sure it is "unable", but current title is a little confusing without any context.